src/d3d9/d3d9_util.cpp

```diff
@@ -41,10 +41,8 @@ namespace dxvk {
           DWORD                     MultisampleQuality,
           VkSampleCountFlagBits*    pCount) {
     uint32_t sampleCount = std::max<uint32_t>(MultiSample, 1u);
-
-    // Check if this is a power of two...
-    if (sampleCount & (sampleCount - 1))
-      return D3DERR_INVALIDCALL;
+    sampleCount = std::min<uint32_t>(MultiSample, 8u);
```

Comment:
Why? Intel for example supports MSAAx16, and I'm pretty sure it can be exposed in d3d9. Also, this makes the max before not do anything, you likely want something like `std::min<uint32_t>(sampleCount, 8u);`.